e2e/playwright.config.ts

```diff
@@ -0,0 +1,37 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+require('dotenv').config();
+
+const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  /* Maximum time one test can run for. */
+  timeout: 120 * 1000,
```

Comment:
any particular reason for the choice of these constants for timeouts, e.g.
- `120 * 1000` (this is 2 mins right)
- `20000` (this is 20 mins?)

Reply:
- 120s cos our test might run for as long as 2 minutes
- 20k is 20s